sagemaker-inference-recommender/huggingface-inference-recommender/huggingface-inference-recommender.ipynb

```diff
@@ -0,0 +1,989 @@
+{
```

Comment:
<p>Optionally add a Conclusion section for summarizing what was accomplished in this notebook and suggesting next steps.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3075/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='74'/>

Reply:
Added Conclusion section with Conclusion and next steps.